recipe.js

```diff
@@ -20,37 +24,114 @@ fetch(randomJokeApi)
     });
 
 //The following function takes the id from the previous page and grabs the corresponding recipe
-function getRecipeById() {
+function getSelectedRecipeById() {
```

Comment:
There seem to be conflicting issues between this and the main branch. You will need to pull main into this branch to resolve issues before pushing again